mreg_cli/exceptions.py

```diff
@@ -58,3 +58,20 @@ class NetworkNotFoundWarning(CliWarning):
     """Warning class for network not found."""
 
     pass
+
+
+class LoginFailedError(CliException):
+    """Error class for login failure."""
+
+    def formatted_exception(self) -> str:
+        """Return a string formatted with 'Login failed:' prefixing the error message.
+
+        :returns: Formatted error message.
+        """
+        return f"Login failed: {super().__str__()}"
+
+    def __str__(self) -> str:
+        """Return the error message."""
+        return "Login failed"
```

Comment:
Could this call `self.formatted_exception` instead for consistency?

```suggestion
        return self.formatted_exception()
```

Reply:
The problem (ish) is that I'm not sure what we should say/leak to the user when logging in fails. This is why I made the default `__str__` be completely vague.

Reply:
Sure. That makes a lot of sense from a security perspective.